package.json

```diff
@@ -1,9 +1,16 @@
 {
-  "name": "lifted-ui",
+  "name": "@liftedinit/ui",
   "private": true,
   "version": "0.1.0",
   "description": "A shared library of components and helpers for Lifted user interfaces",
+  "main": "./dist/index.js",
+  "files": [
+    "dist"
+  ],
   "scripts": {
+    "build": "tsc",
+    "copy-assets": "cp -r ./src/assets ./dist/assets",
+    "prepare": "npm run build && npm run copy-assets",
```

Comment:
When installed from another app, `prepare` runs and both builds the JS version (from the TypeScript) and copies over the static assets.